tests/modules/js-and-css/speculation-rules/speculation-rules-test.php

```diff
@@ -8,10 +8,40 @@
 
 class Speculation_Rules_Tests extends WP_UnitTestCase {
 
-	public function test_plsr_print_speculation_rules() {
-		$output = get_echo( 'plsr_print_speculation_rules' );
+	public function data_provider_to_test_print_speculation_rules(): array {
+		return array(
+			'xhtml' => array(
+				'html5_support' => false,
+			),
+			'html5' => array(
+				'html5_support' => true,
+			),
+		);
+	}
+
+	/**
+	 * @dataProvider data_provider_to_test_print_speculation_rules
+	 */
+	public function test_plsr_print_speculation_rules_without_html5_support( bool $html5_support ) {
+		if ( $html5_support ) {
+			add_theme_support( 'html5', array( 'script' ) );
+		} else {
+			remove_theme_support( 'html5' );
```

Comment:
I don't remember if theme support get reset between tests. Any chance of this causing side effects for other tests?

Reply:
Right. Good point. Fixed in a1037f80